core/translate/pragma.rs

```diff
@@ -410,3 +428,56 @@ fn update_cache_size(
 
     Ok(())
 }
+
+pub const TURSO_CDC_TABLE_NAME: &str = "turso_cdc";
+fn turso_cdc_table_columns() -> Vec<ColumnDefinition> {
+    vec![
+        ast::ColumnDefinition {
+            col_name: ast::Name("operation_id".to_string()),
+            col_type: Some(ast::Type {
+                name: "INTEGER".to_string(),
+                size: None,
+            }),
+            constraints: vec![ast::NamedColumnConstraint {
+                name: None,
+                constraint: ast::ColumnConstraint::PrimaryKey {
+                    order: None,
+                    conflict_clause: None,
+                    auto_increment: false,
+                },
+            }],
+        },
+        ast::ColumnDefinition {
+            col_name: ast::Name("operation_time".to_string()),
+            col_type: Some(ast::Type {
+                name: "INTEGER".to_string(),
+                size: None,
+            }),
+            constraints: vec![],
+        },
+        ast::ColumnDefinition {
+            col_name: ast::Name("operation_type".to_string()),
+            col_type: Some(ast::Type {
+                name: "INTEGER".to_string(),
+                size: None,
+            }),
+            constraints: vec![],
+        },
+        ast::ColumnDefinition {
+            col_name: ast::Name("table_name".to_string()),
+            col_type: Some(ast::Type {
+                name: "TEXT".to_string(),
+                size: None,
+            }),
+            constraints: vec![],
+        },
+        ast::ColumnDefinition {
+            col_name: ast::Name("row_key".to_string()),
```

Comment:
What if the row key is a multi-column primary key like `PRIMARY KEY(a desc, b)` or has some non-binary collation sequence? encoding the key as a blob would require some kind of normalized binary key format that encodes this information. (we might actually benefit from such a normalization scheme anyway in some btree optimizations...)